modules/widgets/simple-payments.php

```diff
@@ -28,29 +55,108 @@ function __construct() {
 				)
 			);
 
+			if ( is_customize_preview() ) {
+				add_action( 'admin_enqueue_scripts', array( $this, 'admin_enqueue_styles_and_scripts' ) );
+
+				add_filter( 'customize_refresh_nonces', array( $this, 'filter_nonces' ) );
+				add_action( 'wp_ajax_customize-jetpack-simple-payments-button-add-new', array( $this, 'ajax_add_new_payment_button' ) );
+			}
+
 			if ( is_active_widget( false, false, $this->id_base ) || is_customize_preview() ) {
 				add_action( 'wp_enqueue_scripts', array( $this, 'enqueue_style' ) );
 			}
 		}
 
-		function enqueue_style() {
-			wp_enqueue_style( 'jetpack-simple-payments-widget-style', plugins_url( 'simple-payments/style.css', __FILE__ ), array(), '20180518' );
-		}
-
 		/**
 		 * Return an associative array of default values.
 		 *
 		 * These values are used in new widgets.
 		 *
 		 * @return array Default values for the widget options.
 		 */
-		public function defaults() {
```

Comment:
Curious about the change here. Was this originally a poor choice, and nothing actually needed access to this instance method?

Reply:
![indeed](https://user-images.githubusercontent.com/233601/41078888-b3f9f89c-69f5-11e8-9cec-60f441cab9f8.gif)

I wouldn't say _poor choice_, more like _automatic writing_.